src/xdocs/config_coding.xml

```diff
@@ -464,15 +464,39 @@ public class A {
           more readable if it comes after the last <code>case</code>.
         </p>
       </subsection>
+        <subsection name="Properties">
+            <table>
+                <tr>
+                    <th>name</th>
+                    <th>description</th>
+                    <th>type</th>
+                    <th>default value</th>
+                </tr>
+                <tr>
+                    <td>skipIfLastAndSharedWithCase</td>
+                    <td>whether to allow <code>default</code> along with case and not last</td>
+                    <td><a href="property_types.html#boolean">Boolean</a></td>
+                    <td>false</td>
+                </tr>
+            </table>
+        </subsection>
 
-      <subsection name="Examples">
-        <p>
-          To configure the check:
-        </p>
-        <source>
+        <subsection name="Examples">
```

Comment:
Please remove the extra 2 spaces added to this section so there is no difference and make sure the true changes stay inline.